lib/plugins/aws/deployFunction/index.test.js

```diff
@@ -153,7 +153,7 @@ describe('AwsDeployFunction', () => {
       const arn = 'arn:aws:iam::123456789012:role/role';
 
       return awsDeployFunction.normalizeArnRole(arn).then((result) => {
-        expect(getAccountIdStub.calledOnce).to.be.equal(false);
+        expect(getAccountInfoStub.calledOnce).to.be.equal(false);
```

Comment:
Use 
```
chai.use(require('chai-as-promised'));
chai.use(require('sinon-chai'));
```
at the top of the file and change all checks in here like this:
```
expect(getAccountInfoStub).to.not.have.been.called;
```

Reply:
The project's lint rules don't like this, but I've included the change. 

```
156:9  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  165:9  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  180:9  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
```